python/dgl/contrib/cugraph/cugraph_utils.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2019-2022, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cugraph
+import cudf
+import dgl
+import cupy
+
+
+# from cugraph to DGL using
+def cugraphToDGL(graph):
+    """
+    Convert fromn a cuGraph graph to a DGLGraph
+    Parameters
+    ----------
+    graph : cugraph graph type {Graph, PropertyGraph}
+        A cuGraph Graph object in GPU memory
+
+    Returns
+    -------
+    g_dgl : DGLGraph
+    """
+    edgelist = graph.edges()
+    src = cupy.asarray(edgelist["src"])
+    dst = cupy.asarray(edgelist["dst"])
+    g_dgl = dgl.graph((src, dst))
+    return g_dgl
+
+
+def dglToCugraph(graph):
```

Comment:
Similarly, let's have this function in DGL. You could put the function in `dgl/convert.py`. Suggested signature:
```python
def to_cugraph(g):
    """Convert a DGL graph to cuGraph object.
    
    The function only converts graph structure while node/edge features are simply dropped.
    Only support homogeneous graph input.

    Parameters
    -----------
    g : DGLGraph
        The input DGL graph.

    Returns
    -------
    cug : cugraph.Graph
        The converted cuGraph object.
    """
```

Reply:
Thanks for the review, Will create a separate PR for this.

Reply:
Started a PR for this https://github.com/dmlc/dgl/pull/4166 here.